### PR TITLE
[MRG+1] Api doc backport no imag build

### PR DESCRIPTION
**Pull request by tacaswell**
backport of #7757  + docs

### CODE REVIEW

File: doc/api/api_changes.rst [hunk 1]
```diff
@@ -201,6 +201,20 @@ replaced by ``nipy_spectral`` and ``nipy_spectral_r`` since matplotlib
 raised a warning. As of matplotlib 2.0.0, using the old names raises a
 deprecation warning. In the future, using the old names will raise an error.
 
+Default install does not include test images
```
Comment (QuLogic) [inline doc/api/api_changes.rst]: does not include -> no longer includes

File: doc/api/api_changes.rst [hunk 2]
```diff
@@ -201,6 +201,20 @@ replaced by ``nipy_spectral`` and ``nipy_spectral_r`` since matplotlib
...
+--------------------------------------------
+
+To reduce the default size of wheels and source installs, the tests
+and baseline images are no longer included by default.
+
+To restore installing the tests and images have a `setup.cfg` with ::
```
Comment (QuLogic) [inline doc/api/api_changes.rst]: images have -> images, use
(or add)

File: doc/devel/contributing.rst [hunk 3]
```diff
@@ -79,10 +79,15 @@ Alternatively, if you do ::
   pip install -v ./
 
 all of the files will be copied to the installation directory however,
-you will have to rerun this command every time the source is changed.
+you will have to rerun this command every time the source is changed.  Additionally you will
+need to copy :file:`setup.cfg.template` to :file:`setup.cfg` and edit to contain ::
```
Comment (QuLogic) [inline doc/devel/contributing.rst]: edit to -> edit it to

**Comment by tacaswell** [comment]
Apparently we have moved to using develop on the master branch, but not in 2.x....

File: doc/api/api_changes.rst [hunk 4]
```diff
@@ -201,6 +201,20 @@ replaced by ``nipy_spectral`` and ``nipy_spectral_r`` since matplotlib
 raised a warning. As of matplotlib 2.0.0, using the old names raises a
 deprecation warning. In the future, using the old names will raise an error.
 
+Default install does not include test images
+--------------------------------------------
+
+To reduce the default size of wheels and source installs, the tests
```
Comment (NelleV) [inline doc/api/api_changes.rst]: Isn't "default" an extra word here?

File: setup.cfg.template [hunk 5]
```diff
@@ -22,10 +22,10 @@
 
 [packages]
 # There are a number of subpackages of matplotlib that are considered
```
Comment (NelleV) [inline setup.cfg.template]: matplotlib -> Matplotlib
(trying to catch them as they come…)

**Comment by tacaswell** [comment]
Took 3 tries, but I think I got all of them.

**Comment by NelleV** [comment]
Thanks!

**Pull request merged**